### DISCUSSION TITLE
doris 从kafka消费数据支持delete操作吗?

### DISCUSSION THREAD

**Discussion by tempestLXC** [apache/doris#15626] [A - General / Q&A]
doris 消费kafka json格式数据, 我测试 新增数据没有问题, 但是update 操作有点问题, delete操作无法同步.

请问是否不支持update和delete操作? 我没在文档上发现相关描述, 想官方确认一下.

另外同步mysql数据现在只有binlog一种吧?😂因为我们业务binlog统一出口是kafka 所以想确认一下这个问题

**Comment by LemonLiTree** [comment]
这个跟doris有什么关系呢？

**Reply by tempestLXC** [reply]
我描述不太准确, 我是想问, doris 从kafka消费数据支持delete操作吗?
